web-api/src/ingestion/routes.rs

```diff
@@ -81,7 +81,7 @@ struct IngestionRequestBody {
     documents: Vec<IngestedDocument>,
 }
 
-#[instrument(skip(state))]
+#[instrument(skip(state, body))]
```

Comment:
:art: since we now skip all arguments, we can instrument it like
```suggestion
#[instrument(skip_all)]
```